### PR TITLE
@spoondla1's pull request for a funny meme

### PR DESCRIPTION
**Pull request by beardofedu**
Modified the content of the my file to include a funny comment.

### CODE REVIEW

**Comment by spoondla1** [comment]
got the information.

File: _posts/0000-01-19-glerby.md [hunk 1]
```diff
@@ -6,4 +6,4 @@ title: "Welcome to our slide deck!"
 ![puss-in-boots](https://cloud.githubusercontent.com/assets/16547949/25401020/740dd682-29c2-11e7-9906-ac5c3a8496c7.jpg)
 {: .center}
 
-Caption here
+able to get the class
```
Comment (Momoko-Sanchez) [inline _posts/0000-01-19-glerby.md:9]: looks good! 👍

**Review by Momoko-Sanchez — COMMENTED** [review]
Here goes the summary, everything okay.

**Review by Momoko-Sanchez — APPROVED** [review]
everything great!

**Comment by aninda-mukherjee** [comment]
hello hello hello

**Pull request merged**